src/containers/articles/ShareArticle.js

```diff
@@ -0,0 +1,158 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { apiCalls } from "../../Helpers/axios";
+import Input from "../../components/Input/Input";
+import Modal from "../../components/Modal/Modal";
+import { ShowMessage } from "../../redux/actions/SnackBarAction";
+import "../../assets/css/sharing.css";
+
+export class ShareBar extends Component {
+  state = {
+    openmodal: false,
+    email: ""
+  };
+  handleSuccessShareResponse = ({ data }) => {
+    const newLink = data.shared_link.replace(
+      "http://ah-premier-staging.herokuapp.com/api",
+      "https://ah-fortem-staging.herokuapp.com"
```

Comment:
Make these URLs dynamic. Have it in the `.env` file. If in future the domains change, you wouldn't have to go back to update the codebase.

Reply:
Thank you for catching that. I have resolved this